obspy/io/dmx/core.py

```diff
@@ -0,0 +1,137 @@
+"""
+INGV DMX bindings to ObsPy core module.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+from tempfile import SpooledTemporaryFile
+from obspy.core.util.attribdict import AttribDict
+from obspy import Stream, Trace, UTCDateTime
+import numpy as np
```

Comment:
ordering of imports..
 - future
 - <space>
 - builtins (here tempfile)
 - <space>
 - third party (numpy etc)
 - <space>
 - obspy
 - <space>
 - relative obspy imports (none here)

in each block first `import ...` and then `from ... import ...` statements, alphabetically inside blocks

so:

```
from __future__ import (absolute_import, division, print_function,
                        unicode_literals)
from future.builtins import *  # NOQA

from tempfile import SpooledTemporaryFile

import numpy as np

from obspy import Stream, Trace, UTCDateTime
from obspy.core.util.attribdict import AttribDict
```

Reply:
ok boss